usecases/no2-monitoring/index.md

```diff
@@ -25,44 +59,325 @@ extent = {
+from pandas import Series
+import numpy as np
+
+N = 31
+def apply_timeseries(series: Series, context: dict) -> Series:
+  return np.convolve(series, np.ones(N)/N, mode='same')
+```
+:::
+
+### 4. What do you want to know?
+
+Now it's time to decide what we actually want to compute and get an insight into.
+
+Currently, the data cube at still has 4 dimensions:
+The spatial dimensions `x` and `y` covering the extent of Münster,
+the temporal dimension `t` covering usually about 365 values of the given year, and
+the band dimension `bands` with just a single label `NO2`.
+
+As the `bands` dimension only has a single label it gets dropped automatically during export, 
+which means we now basically have a series of NO₂ maps, one for each day.
+We have several options now:
+
+1. Store the daily maps as netCDF file
+2. Reduce the temporal dimension to get a single map for the year as GeoTIff
+3. Reduce the spatial dimensions to get a timeseries as JSON
+
+#### 4.1 netCDF Export
+
+To store daily maps in a netCDF file, we simply need to store the datacube:
+
+<CodeSwitcher :languages="{py: 'Python', js: 'JavaScript', r: 'R'}">
+<template v-slot:py>
+
+```python
+datacube = datacube.save_result(format = "netCDF")
+```
+
+</template>
+<template v-slot:js>
+
+```js
+datacube = builder.save_result(datacube, "netCDF");
+```
+
+</template>
+<template v-slot:r>
+
+```r
+datacube = p$save_result(data = datacube, format = "netCDF")
+```
+
+</template>
+</CodeSwitcher>
+
+#### 4.2 Map for a year as GeoTiff
+
+To get a map with values for a year we need to reduce the temporal dimensions by reducing the values along the temporal dimension.
+You can run different reducers such as `mean` (in the example below), `max` or `min` so that
+we get a single value for each pixel.
+
+Lastly, you need to specify that you want to store the result as GeoTiff (`GTiff` due to the naming in GDAL).
+
+<CodeSwitcher :languages="{py: 'Python', js: 'JavaScript', r: 'R'}">
+<template v-slot:py>
+
+```python
+datacube = datacube.reduce_dimension(reducer = "mean", dimension = "t")
+datacube = datacube.save_result(format = "GTiff")
+```
+
+</template>
+<template v-slot:js>
+
+```js
+let reducer = (data, _, builder) => builder.mean(data);
+datacube = builder.reduce_dimension(datacube, reducer, dimension = "t");
+datacube = builder.save_result(datacube, "GTiff");
+```
+
+</template>
+<template v-slot:r>
+
+```r
+reducer = function(data, context) { 
+  p$mean(data = data)
+}
+datacube = p$reduce_dimension(data = datacube, reducer = reducer, dimension = "t")
+datacube = p$save_result(data = datacube, format = "GTiff")
+```
+
+</template>
+</CodeSwitcher>
+
+
+#### 4.3 Timeseries as JSON
+
+To get a timeseries we need to reduce the spatial dimensions by aggregating them.
+You can run different aggregation methods such as `mean` (in the example below), `max` or `min` so that
+we get a single value for each timestep.
+
+Lastly, you need to specify that you want to store the result as JSON.
```

Comment:
Can you specify also a different file format? CSV as previous UC have done?

Reply:
I haven't tried yet...